activities/Moon.activity/css/activity.css

```diff
@@ -111,7 +111,7 @@ canvas {
 }
 
 .tutorial-next-icon2 {
-	background-image: url(../icons/go-right.svg);
+	background-image: url(../../../icons/go-right.svg);
```

Comment:
Not at all a good idea to rely on icons in Sugarizer Core. Each activity should be deployable by itself